src/main/java/jhoselineteran/assigment2/CollatzConjecture.java

```diff
@@ -0,0 +1,75 @@
+package jhoselineteran.assigment2;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Esta clase implementa la Conjetura de Collatz
```

Comment:
Please don't add anything in Spanish. Fix the indentation level in your classes.